app/machinedriver_data.go

```diff
@@ -66,8 +66,8 @@ func addMachineDrivers(management *config.ManagementContext) error {
 	if dl := os.Getenv("CATTLE_DEV_MODE"); dl != "" {
 		linodeBuiltin = false
 	}
-	if err := addMachineDriver(nodetemplate.Linodedriver, "https://github.com/linode/docker-machine-driver-linode/releases/download/v0.1.7/docker-machine-driver-linode_linux-amd64.zip",
-		"/assets/rancher-ui-driver-linode/component.js", "faaf1d7d53b55a369baeeb0855b069921a36131868fe3641eb595ac1ff4cf16f", []string{"api.linode.com"}, linodeBuiltin, linodeBuiltin, management); err != nil {
+	if err := addMachineDriver(nodetemplate.Linodedriver, "https://github.com/linode/docker-machine-driver-linode/releases/download/v0.1.8/docker-machine-driver-linode_darwin-amd64.zip",
+		"/assets/rancher-ui-driver-linode/component.js", "fb5b7486afd0a5ddea72d72dbe8fba941fb28236130323a88bf0c5c87336d3d1", []string{"api.linode.com", "linode.github.io"}, linodeBuiltin, linodeBuiltin, management); err != nil {
```

Comment:
Is it a typo `linode_darwin-amd64`? I guess maybe we should use `driver-linode_linux-amd64`.